Sources/SourceKitLSP/Swift/DocumentSymbols.swift

```diff
@@ -238,3 +299,15 @@ fileprivate extension TokenKind {
     }
   }
 }
+
+fileprivate extension String {
+  /// Compatibility function of the `trimmingPrefix` function in the stdlib. We should be able to remove this once we
+  /// only support macOS 13+
+  func trimmingPrefix(while condition: (Character) -> Bool) -> Substring {
+    var result = self[...]
+    while let first = result.first, condition(first) {
+      result = result[result.index(after: result.startIndex)...]
```

Comment:
```suggestion
      result = result.dropFirst()
```

Reply:
Also given my comment above, you could add the other version of `trimmingPrefix` that takes a `Sequence`

Reply:
I just removed it and am now trimming `/`, `*` and whitespace on either side of the comment.